code/game/objects/items/oddities.dm

```diff
@@ -0,0 +1,211 @@
+		STAT_THG = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_landscape
+	name = "alien landscape photo"
+	desc = "There some ire about this planet."
+	icon_state = "photo_landscape"
+	oddity_stats = list(
+		STAT_COG = 5,
+		STAT_THG = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_coridor
+	name = "surreal maint photo"
+	desc = "Something wrong, yet so familiar about this place."
+	icon_state = "photo_corridor"
+	oddity_stats = list(
+		STAT_MEC = 5,
+		STAT_THG = 5,
+	)
+
+/obj/item/weapon/oddity/common/photo_eyes
+	name = "observer photo"
+	desc = "What the fuck is this even."
+	icon_state = "photo_corridor"
+	oddity_stats = list(
+		STAR_ROB = 6,
+		STAT_THG = 6,
+		STAT_VIG = 6,
+	)
+
+/obj/item/weapon/oddity/common/old_newspaper
+	name = "old newspaper"
+	desc = "It contains a report on some old and strange phenomenon. Maybe it's lies, maybe it's corporate experements gone wrong."
+	icon_state = "old_newspaper"
+	oddity_stats = list(
+		STAT_MEC = 4,
+		STAT_COG = 4,
+		STAT_BIO = 4,
+	)
+
+/obj/item/weapon/oddity/common/paper_crumpled
+	name = "turn-out page"
+	desc = "This is ALMOST makes sense."
+	icon_state = "paper_crumpled"
+	oddity_stats = list(
+		STAT_MEC = 6,
+		STAT_COG = 6,
+		STAT_BIO = 6,
+	)
+
+/obj/item/weapon/oddity/common/paper_omega
+	name = "collection of obscure reports"
+	desc = "Even the authors seem to be rather skeptical about their findings. Yet they are not connected to each other, but results are simular."
+	icon_state = "paper_omega"
+	oddity_stats = list(
+		STAT_MEC = 8,
+		STAT_COG = 8,
+		STAT_BIO = 8,
+	)
+
+/obj/item/weapon/oddity/common/book_eyes
+	name = "observer book"
+	desc = "This book details the information on some cyber creatures. Who did this, how this is even possible?"
+	icon_state = "book_eyes"
+	oddity_stats = list(
+		STAR_ROB = 9,
+		STAT_THG = 9,
+		STAT_VIG = 9,
+	)
+
+/obj/item/weapon/oddity/common/book_omega
+	name = "occult book"
+	desc = "That's mostly does not make any logical sense. However, those stories are at least interesting."
+	icon_state = "book_omega"
+	oddity_stats = list(
+		STAT_BIO = 6,
+		STAR_ROB = 6,
+		STAT_VIG = 6,
+	)
+
+/obj/item/weapon/oddity/common/book_bible
+	name = "old bible"
+	desc = "Oh, how quickly we forgot."
+	icon_state = "book_bible"
+	oddity_stats = list(
+		STAR_ROB = 5,
+		STAT_VIG = 5,
+	)
+
+/obj/item/weapon/oddity/common/old_money
+	name = "old_money"
+	desc = "It's not like organization that issues this exist now."
+	icon_state = "old_money"
+	oddity_stats = list(
+		STAT_ROB = 4,
+		STAT_THG = 4,
+	)
+
+/obj/item/weapon/oddity/common/healthscanner
+	name = "odd healthscanner"
+	desc = "It's broken and stuck on some really strange readings. Was this even human?"
+	icon_state = "healthscanner"
+	item_state = "electronic"
+	oddity_stats = list(
+		STAT_COG = 8,
+		STAT_BIO = 8,
+	)
+
+/obj/item/weapon/oddity/common/old_pda
+	name = "broken pda"
+	desc = "This is old, Nanotrasen era PDA, issued to employes of this corporation all over the galaxy."
```

Comment:
"An old Nanotrasen era PDA. These were issued to their employees all throughout the galaxy."